backend/src/main/java/quarano/actions/quarantine/QuarantineEndChecker.java

```diff
@@ -0,0 +1,50 @@
+package quarano.actions.quarantine;
+
+import static quarano.department.TrackedCase.Status.OPEN;
+
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.joda.time.DateTime;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import quarano.actions.ActionItemRepository;
+import quarano.department.TrackedCase;
+import quarano.department.TrackedCaseRepository;
+
+@RequiredArgsConstructor
+@Slf4j
+@Component
+public class QuarantineEndChecker {
+
+	private final TrackedCaseRepository trackedCases;
+	private final ActionItemRepository items;
+
+	@Scheduled(cron = "0 8 * * *")
```

Comment:
How often do we want to check for ending quarantines? Once a day, or more often?